packages/popover/src/uxp-popover.css

```diff
@@ -32,14 +32,16 @@ governing permissions and limitations under the License.
 }
 
 /* Workaround for - https://jira.corp.adobe.com/browse/UXP-21332 */
-:host([placement*='left']) .tip {
-    transform: rotate(-90deg);
-    height: var(--sp-popover-tip-size);
-    margin-left: var(--uxp-swc-placement-left-margin-left);
+:host([placement*='right']) .tip {
```

Comment:
Don't we need styles for placement:left?

Reply:
Yes, I will remove unused variables.